src/app/mosh_prefs.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
```

Comment:
I wrote the HTML and took the CSS from nassh. I'll split them into separate files so the distinction is clean.